### ISSUE TITLE
Error using this image: fabric8/java-main - must specify a user that is numeric and within the range of allowed users

### ISSUE THREAD

**Issue by rafabene** [fabric8io/java-main#13]
When I try to use this builder image I get the following error: F0318 11:56:04.827924       1 builder.go:204] Error: build error: image "fabric8/java-main@sha256:aaf9568dcb0923484f466d1a81e7a898006a934ab437f20fffe65b1b2dd1fdce" must specify a user that is numeric and within the range of allowed users

**Comment by christian-posta** [comment]
Use one of these ones instead?

https://github.com/fabric8io/base-images/tree/master/java/images
https://github.com/fabric8io/base-sti

On Fri, Mar 18, 2016 at 9:02 AM, Rafael Benevides notifications@github.com
wrote:

> When I try to use this builder image I get the following error: F0318
> 11:56:04.827924 1 builder.go:204] Error: build error: image
> "fabric8/java-main@sha256:aaf9568dcb0923484f466d1a81e7a898006a934ab437f20fffe65b1b2dd1fdce"
> must specify a user that is numeric and within the range of allowed users
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/fabric8io/java-main/issues/13

## 

_Christian Posta_
twitter: @christianposta
http://www.christianposta.com/blog
http://fabric8.io

**Comment by rafabene** [comment]
Thanks. I'll try it.

On Fri, Mar 18, 2016 at 12:18 PM, Christian Posta notifications@github.com
wrote:

> Use one of these ones instead?
> 
> https://github.com/fabric8io/base-images/tree/master/java/images
> https://github.com/fabric8io/base-sti
> 
> On Fri, Mar 18, 2016 at 9:02 AM, Rafael Benevides <
> notifications@github.com>
> wrote:
> 
> > When I try to use this builder image I get the following error: F0318
> > 11:56:04.827924 1 builder.go:204] Error: build error: image
> > "fabric8/java-main@sha256
> > :aaf9568dcb0923484f466d1a81e7a898006a934ab437f20fffe65b1b2dd1fdce"
> > must specify a user that is numeric and within the range of allowed users
> > 
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/fabric8io/java-main/issues/13
> 
> ## 
> 
> _Christian Posta_
> twitter: @christianposta
> http://www.christianposta.com/blog
> http://fabric8.io
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/fabric8io/java-main/issues/13#issuecomment-198432738

## 

_Rafael Benevides | Senior Software Engineer_
JBoss Developer Materials lead
M: +1-919-592-6255

Better technology. Faster innovation. Powered by community collaboration.
See how it works at www.redhat.com

**Comment by rafabene** [comment]
In fact I was looking for an Openshift S2I Java image. Those images doesn't
seem to be possible to be used as builder images.

On Fri, Mar 18, 2016 at 2:41 PM, Rafael Benevides benevides@redhat.com
wrote:

> Thanks. I'll try it.
> 
> On Fri, Mar 18, 2016 at 12:18 PM, Christian Posta <
> notifications@github.com> wrote:
> 
> > Use one of these ones instead?
> > 
> > https://github.com/fabric8io/base-images/tree/master/java/images
> > https://github.com/fabric8io/base-sti
> > 
> > On Fri, Mar 18, 2016 at 9:02 AM, Rafael Benevides <
> > notifications@github.com>
> > wrote:
> > 
> > > When I try to use this builder image I get the following error: F0318
> > > 11:56:04.827924 1 builder.go:204] Error: build error: image
> > > "fabric8/java-main@sha256
> > > :aaf9568dcb0923484f466d1a81e7a898006a934ab437f20fffe65b1b2dd1fdce"
> > > must specify a user that is numeric and within the range of allowed
> > > users
> > > 
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/fabric8io/java-main/issues/13
> > 
> > ## 
> > 
> > _Christian Posta_
> > twitter: @christianposta
> > http://www.christianposta.com/blog
> > http://fabric8.io
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/fabric8io/java-main/issues/13#issuecomment-198432738
> 
> ## 
> 
> _Rafael Benevides | Senior Software Engineer_
> JBoss Developer Materials lead
> M: +1-919-592-6255
> 
> Better technology. Faster innovation. Powered by community collaboration.
> See how it works at www.redhat.com

## 

_Rafael Benevides | Senior Software Engineer_
JBoss Developer Materials lead
M: +1-919-592-6255

Better technology. Faster innovation. Powered by community collaboration.
See how it works at www.redhat.com

**Comment by markvgo** [comment]
I am getting this same error,  did you find out why

**Comment by rafabene** [comment]
Nope! @markvgo I just used another image: https://github.com/redhat-helloworld-msa/aloha/blob/master/Dockerfile#L1

**Comment by iocanel** [comment]
@rafabene, @markvgo, @christian-posta: this project has moved here: https://github.com/fabric8io-images/s2i.

**Comment by markvgo** [comment]
Thanks 

On 07/06/2016 01:56, Ioannis Canellos wrote: 

> @rafabene
> [1], @markvgo [2], @christian-posta [3]: this project has moved here:
> https://github.com/fabric8io-images/s2i [4]. 
> 
> ## 
> 
> You are
> receiving this because you were mentioned.
> Reply to this email
> directly, view it on GitHub [5], or mute the thread [6].

## Links:

[1] https://github.com/rafabene
[2]
https://github.com/markvgo
[3] https://github.com/christian-posta
[4]
https://github.com/fabric8io-images/s2i
[5]
https://github.com/fabric8io/java-main/issues/13#issuecomment-224002922
[6]
https://github.com/notifications/unsubscribe/AHMmRZvq0qU2lxbNhjyU9mNKs-BWQtlFks5qJEMigaJpZM4Hz_ZG
